BUILD

```diff
@@ -77,7 +77,31 @@ linux_packages(
     depends = [
         "openjdk-8-jre"
     ],
-    modes = {
+    permissions = {
+        "./var/log/grakn/": "0777",
+    },
+    symlinks = {
+        "./usr/local/bin/grakn": "/opt/grakn/core/grakn",
+        "./opt/grakn/core/logs": "/var/log/grakn/",
+    },
+)
+
+
+rpm_package(
+    name = "deploy-rpm",
```

Comment:
`name = "distribution-rpm"`

Reply:
@lolski as discussed on Slack, it's already named correctly